[Ruby] New Concept exercise: numbers (#1295)
exercism/v3: languages/ruby/exercises/concept/numbers/.docs/hints.md

### General

> - [The Ruby `Float` class][float-ruby]

Reviewer: I'd be slightly hesitant to point to the official Ruby docs as they can be quite overwhelming. My preference would be to link to specific methods in the task-specific hints (without giving away the actual name of the method in this file).

- [The Ruby `Integer` class][integer-ruby]
- [`if-else-unless` statements][if-else-unless]

### 1. Calculate the production rate per second

- Determining the success rate can be done through a [conditional statement][if-else-unless].
- Multiplication can be done between instances of `Integer` and `Float`. The result will always be an instance of `Float`.
- Numbers can be compared using the built-in [comparison-operators][comparison-operators].

### 2. Calculate the number of working items produced per second

- The [`Float`][float-ruby] class implements a method to convert it's instances into instances of `Integer`.

> - The [`Integer`][integer-ruby] class implements a method to convert it's instances into instances of `Float`.

Author: "convert" is probably the wrong word here, probably should be replaced but I can't come up with anything

Reviewer 2: I would not use the `convert` wording, as it really doesn't.  I might just say:
```suggestion
- The [`Float`][float-ruby] class implements a method to return an instance of `Integer`.
- The [`Integer`][integer-ruby] class implements a method to return an instance of `Float`.
```

Author: Sounds better

Reviewer 2: And is truthful. ;)
